gammapy/time/tests/test_lightcurve.py

```diff
@@ -216,3 +216,25 @@ def test_lightcurve_interval_maker():
     assert len(intervals) == 9
     t = intervals[0]
     assert_allclose(t[1].value - t[0].value, 500 / (24 * 3600), rtol=1e-5)
+
+
+@requires_data('gammapy-extra')
+@requires_dependency('scipy')
+def test_lightcurve_adaptative_interval_maker():
+    spec_extract = spec_extraction()
+    lc_estimator = LightCurveEstimator(spec_extract)
+    separator = [(53343.94050200008 + 53343.952979345195) / 2]
```

Comment:
IMO you can see here that just using floats as `separator` leads to slightly less readable code in callers.
You have:
```
separator = [(53343.94050200008 + 53343.952979345195) / 2]
```
If you use `Time` objects and change the parameter name to include the word "time", it would look like this:
```
time_split_points = Time([(53343.94050200008 + 53343.952979345195) / 2], scale='tt', format='mjd')
```
or you could directly work with more human-readable times in the tests if you like.